xbmc/pvr/guilib/PVRGUIChannelIconUpdater.cpp

```diff
@@ -48,13 +48,14 @@ void CPVRGUIChannelIconUpdater::SearchAndUpdateMissingChannelIcons() const
   CLog::Log(LOGINFO, "Starting PVR channel icon search");
 
   // create a map for fast lookup of normalized file base name
-  std::map<std::string, std::string> fileItemMap;
+  using FileNamesMap = std::map<std::string, std::string, std::less<>>;
+  FileNamesMap fileItemMap;
   for (const auto& item : fileItemList)
   {
     std::string baseName = URIUtils::GetFileName(item->GetPath());
     URIUtils::RemoveExtension(baseName);
     StringUtils::ToLower(baseName);
-    fileItemMap.insert({baseName, item->GetPath()});
+    fileItemMap.try_emplace(baseName, item->GetPath());
```

Comment:
Possible optimization:
```suggestion
    fileItemMap.try_emplace(std::move(baseName), item->GetPath());
```